packages/styled/types/tests.tsx

```diff
@@ -86,6 +86,7 @@ const Button = styled.button``
 const Input = styled.input`
   & + ${Label}: {
     margin-left: 3px;
+    color: ${props => props.theme.primary};
```

Comment:
Have you confirmed that this would be broken without your fix?

Reply:
Yep! definetely broken before and fix after 😉